src/constants/typography.js

```diff
@@ -33,64 +34,51 @@ const textStyles = {
   uppercase: "uppercase",
   lowercase: "lowercase",
   capitalize: "capitalize",
+  underline: "underline",
```

Comment:
I think in some of the spots we might want to use bottom padding with a border bottom for custom underlines

Reply:
Got it! I just added a const for that in the types and used it for the connect link. The screenshot below is bottom-padding 0 but we can always change the style
![connect](https://user-images.githubusercontent.com/60420625/113613581-6c4bed80-961f-11eb-9d02-a207daeccc6d.jpg)